Fix meta descriptions (#9399)
pantheon-systems/documentation: source/content/certification/study-guide-cms/09-automate.md

---
title: WebOps Certification
subtitle: "Chapter 9: Additional Automation"

> description: "Learn how to leverage build steps on Pantheon to run Integrated Composer workflows."

Reviewer: ```suggestion
description: "Learn how to leverage Integrated Composer and Autopilot"
```
